bitnami/moodle/4.4/debian-12/rootfs/post-init.d/sql-mysql.sh

```diff
@@ -33,6 +33,15 @@ mysql_execute() {
     return "$failure"
 }
 
+MARIADB_HOST=${MARIADB_HOST:-$MOODLE_DATABASE_HOST}
+MARIADB_PORT_NUMBER=${MARIADB_PORT_NUMBER:-$MOODLE_DATABASE_PORT_NUMBER}
+MARIADB_ROOT_USER=${MARIADB_ROOT_USER:-$MOODLE_DATABASE_USER}
+MARIADB_DATABASE=${MARIADB_DATABASE:-$MOODLE_DATABASE_NAME}
+if [[ "${ALLOW_EMPTY_PASSWORD:-no}" != "yes" ]]; then
+  MARIADB_ROOT_PASSWORD=${MARIADB_ROOT_PASSWORD:-$MOODLE_DATABASE_PASSWORD}
+fi
```

Comment:
I would move this block to line 20, above the function `mysql_execute`.

Reply:
Yeah I see your point. This has now been changed